CMakeLists.txt

```diff
@@ -104,5 +106,6 @@ set_target_properties(tests PROPERTIES
 	VERSION ${PROJECT_VERSION}
 	CXX_STANDARD 17)
 
+target_include_directories(tests PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/deps/plog/include)
```

Comment:
You should instead set the two other `target_include_directories` as `PUBLIC`, the include will then be propagated by CMake with the library.